containers/py38/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM alpine
 # finally it removes the cache - all in one go to prevent another layer which we don't need
 RUN set -ex && \
 # install python, pip,  and pytest \
-    apk add --no-cache python3=3.8.7-r1 && \
+    apk add --no-cache python3=3.8.8-r0 && \
```

Comment:
This is getting old pretty fast I think, I wonder if we could relax this a bit to prevent changing this every time there is a PR in this repo

Reply:
yeah the hadolint is annoying